### ISSUE TITLE
Public portal for publishing/browsing 5s-crate

### ISSUE THREAD

**Issue by stain** [federated-analytics/technical-roadmap#22]

**Comment by elichad** [comment]
Related issues: https://github.com/eScienceLab/RO-Crate_Issues/issues/172 and https://github.com/eScienceLab/RO-Crate_Issues/issues/165

**Comment by prquinlan** [comment]
@elichad  and @stain  still planned?

**Comment by elichad** [comment]
Yes - see https://github.com/eScienceLab/RO-Crate_Issues/issues/172 (dashboard of many crates, for usage statistics and such) and https://github.com/eScienceLab/RO-Crate_Issues/issues/165 (for pulling Five Safes information from individual crates)

See https://profiles.ro-crate.org/en (beta) and https://cebeme.manchester.ac.uk/en/ for examples of the dashboard platform we plan to use (Sampo-UI). The former is using RO-Crate metadata directly.

**Comment by prquinlan** [comment]
Thank you @elichad  - is this under HDR UK?
